test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
I'd just remove your test directory and install rspec. `rm -rf test` from the root of your project. Then follow the instructions in the rspec-rails README.